sql/driving_distance/withPointsDD.sql

```diff
@@ -23,41 +23,42 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  ********************************************************************PGR-GNU*/
 
 -- SINGLE
---v2.6
+--v3.6
 CREATE FUNCTION pgr_withPointsDD(
     TEXT,   --edges_sql (required)
     TEXT,   -- points_sql (required)
     BIGINT, -- from_vid (required)
     FLOAT,  -- distance (required)
 
     directed BOOLEAN DEFAULT true,
-    driving_side CHAR DEFAULT 'b',
+    driving_side CHAR DEFAULT 'r',
     details BOOLEAN DEFAULT false,
 
     OUT seq INTEGER,
+    OUT start_vid BIGINT,
     OUT node BIGINT,
     OUT edge BIGINT,
     OUT cost FLOAT,
     OUT agg_cost FLOAT)
 RETURNS SETOF RECORD AS
 $BODY$
-    SELECT seq, node, edge, cost, agg_cost
+    SELECT seq, start_vid, node, edge, cost, agg_cost
     FROM _pgr_withPointsDD(_pgr_get_statement($1), _pgr_get_statement($2), ARRAY[$3]::BIGINT[], $4, $5, $6, $7, false);
 $BODY$
 LANGUAGE SQL VOLATILE STRICT
 COST 100
 ROWS 1000;
 
 -- MULTIPLE
---v2.6
+--v3.6
 CREATE FUNCTION pgr_withPointsDD(
     TEXT,     --edges_sql (required)
     TEXT,     -- points_sql (required)
     ANYARRAY, -- from_vid (required)
     FLOAT,    -- distance (required)
 
     directed BOOLEAN DEFAULT true,
-    driving_side CHAR DEFAULT 'b',
+    driving_side CHAR DEFAULT 'r',
```

Comment:
This is a tricky change, 
Basically the default for versions 3.5 & under is `b`and the default for 3.6 and after is `r`
For the moment lest leave it like it is here, but open an issue "reminder of default value" in your fork and put a link to this comment

Reply:
issue for reminder: https://github.com/squarege/GSoC-pgRouting/issues/5

Reply:
See #300